### PR TITLE
Update tool to 2.0.10

### PR DESCRIPTION
**Pull request by ljmf00**
Fixes https://github.com/codacy/codacy-meta/issues/23 .

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,171 +1,60 @@
...
-      - sbt_dependencies:
+      - codacy/checkout_and_version:
+          write_sbt_version: true
+      - codacy/sbt:
+          name: populate_cache
+          cmd: |
+            sbt "set scalafmtUseIvy in ThisBuild := false;
+                 set name := \"$CIRCLE_PROJECT_REPONAME\";
+                 update"
+          persist_to_workspace: true
```
Comment (lolgab) [inline .circleci/config.yml]: You can shrink this step in the other as we are doing for all the other tools

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,18 @@
-FROM codacy-flawfinder-base:latest
+FROM library/openjdk:8-jre-alpine
 
-LABEL maintainer="team@codacy.com"
+ARG toolVersion
 
-RUN adduser -u 2004 -D docker
-WORKDIR /opt/docker
+RUN \
+    apk add --no-cache bash python2 && \
```
Comment (lolgab) [inline Dockerfile]: Can you check if we need python2 or we can upgrade to python3?

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,9 @@
+all: publish-base generate-docs
+
+publish-base:
+	docker build --no-cache -t "codacy-flawfinder-base:latest" -f Dockerfile . --build-arg toolVersion="$(shell cat .flawfinder-version | tr -d '\n')"
+
+generate-docs:
+	docker run -i codacy/codacy-flawfinder-base:latest -QD --listrules > .tmp_errorlist
+	sbt "runMain codacy.flawfinder.DocGenerator .tmp_errorlist"
+	rm -rf .tmp_errorlist
```
Comment (lolgab) [inline Makefile]: Newline here

File: .circleci/config.yml (hunk 1 above)
Comment (ljmf00) [reply .circleci/config.yml]: Done

File: Dockerfile (hunk 2 above)
Comment (ljmf00) [reply Dockerfile]: Done

File: Makefile (hunk 3 above)
Comment (ljmf00) [reply Makefile]: Done

File: .circleci/config.yml [hunk 4]
```diff
@@ -1,171 +1,53 @@
...
+            - plugins_test
           filters:
             branches:
               only:
                 - master
+      - codacy/tag_version:
+          name: tag_version
+          context: CodacyAWS
+          requires:
+            - publish_dockerhub
```
Comment (lolgab) [inline .circleci/config.yml]: A newline here

File: .gitignore [hunk 5]
```diff
@@ -7,3 +7,6 @@
 .idea
 .vscode
 .DS_Store
+
+.bloop
+.metals
```
Comment (lolgab) [inline .gitignore]: A newline here

File: Dockerfile [hunk 6]
```diff
@@ -1,13 +1,18 @@
-FROM codacy-flawfinder-base:latest
+FROM library/openjdk:8-jre-alpine
```
Comment (lolgab) [inline Dockerfile]: here use `openjdk:8-jre-alpine` image

File: Dockerfile [hunk 7]
```diff
@@ -1,13 +1,18 @@
...
 
-COPY --chown=docker:docker "target/docker/stage/opt/docker" "/opt/docker"
-COPY --chown=docker:docker src/main/resources/docs /docs
-
-USER docker
-ENTRYPOINT ["bin/codacy-flawfinder"]
-CMD []
\ No newline at end of file
+ENTRYPOINT ["/usr/bin/flawfinder"]
+CMD []
```
Comment (lolgab) [inline Dockerfile]: This should be implicit, no need to write `CMD []`

File: project/plugins.sbt [hunk 8]
```diff
@@ -1 +1,4 @@
-addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.6")
+resolvers += Resolver.jcenterRepo
+addSbtPlugin("com.codacy" % "codacy-sbt-plugin" % "17.1.2")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.3.15")
```
Comment (lolgab) [inline project/plugins.sbt]: A newline here

File: .circleci/config.yml (hunk 4 above)
Comment (ljmf00) [reply .circleci/config.yml]: Done

File: .gitignore (hunk 5 above)
Comment (ljmf00) [reply .gitignore]: Done

File: Dockerfile (hunk 6 above)
Comment (ljmf00) [reply Dockerfile]: Done

File: Dockerfile (hunk 7 above)
Comment (ljmf00) [reply Dockerfile]: Done

File: project/plugins.sbt (hunk 8 above)
Comment (ljmf00) [reply project/plugins.sbt]: Done

**Pull request merged**